### DISCUSSION TITLE
[Mobile] On ios, Select bold and enter Chinese characters, which does not take effect

### DISCUSSION THREAD

**Discussion by lsfern** [singerdmx/flutter-quill#552] [Mobile]
My issue is about [Mobile]

On ios, select bold and enter Chinese characters, which does not take effect

**Comment by OldJii** [comment]
Not only the bold effect, all functions will be invalid after clicking the Chinese prompt of the ios default input method.

**Comment by fuwuyuan** [comment]
How can us solve this？

**Comment by singerdmx** [comment]
I wish I could help but I have no idea about it

**Reply by mny459** [reply]
Repreduce step:
1. On iOS device(not Simulator, this is mportant), use the Chinese Input Method
2. Input any letter like 'H', and I get debug message below:
![image](https://user-images.githubusercontent.com/24559456/185161848-e17604ca-0dbf-425a-9a4c-0679588e968e.png)

3. Then I select '哈', I get debug message below:
![image](https://user-images.githubusercontent.com/24559456/185162263-8c43db26-be30-4c00-82ee-cece4d4c2cc5.png)

Is that "delete" operation also delete Bold attr?

**Comment by 1252158112** [comment]
already fixed
